wandb/sdk/wandb_run.py

```diff
@@ -3005,6 +3005,11 @@ def use_artifact(
                 project_name=r.project,
                 use_as=use_as or artifact_or_name,
             )
+            if aliases is None:
+                aliases = []
+            elif isinstance(aliases, str):
```

Comment:
Since the input is `aliases: Optional[List[str]]` should we not also handle the case when a user passes in:
`wandb.run.use_artifact("mnist:v0", aliases=["my_alias_3", "my_alias_4", ""my_alias_5"])`?
For both here and beneath in line 3017